packages/react-composites/src/composites/CallComposite/styles/CallConfiguration.styles.ts

```diff
@@ -91,10 +91,10 @@ export const configurationSectionStyle: IStackStyles = {
 /**
  * @private
  */
-export const selectionContainerStyle = (theme: ITheme): string =>
+export const selectionContainerStyle = (theme: ITheme, browser?: string): string =>
```

Comment:
nit: difficult to understand why safari means the height is different, suggestion: use something like `noSpeakerDropdownShown` as the variable name here